packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -54,6 +58,82 @@ List<String> debugDescribeTransform(Matrix4 transform) {
   return matrix;
 }
 
+bool _extensionsInitialized = false;
+
+void initServiceExtensions() {
+  if (_extensionsInitialized)
+    return;
+
+  _extensionsInitialized = true;
+
+  assert(() {
+    registerExtension('flutter', _flutter);
+    // TODO: Expose debugDumpApp().
+    // TODO: Expose StatisticsOverlay.
+    registerExtension('flutter.debugPaint', _debugPaint);
+    registerExtension('flutter.timeDilation', _timeDilation);
+
+    // Emit an info level log message; this tells the debugger that the Flutter
+    // service extensions are registered.
+    log('Flutter initialized', name: 'flutter', level: 800);
+
+    return true;
+  });
+}
+
+/// Just respond to the request. Clients can use the existence of this call to
+/// know that the debug client is a Flutter app.
```

Comment:
I think having this in dart:developer itself is a better way to go. Solves the problem more comprehensively.